src/SandboxSizeParameters.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Sandboxes\Api;
+
+class SandboxSizeParameters
+{
+    private ?int $storageSize_GB;
+
+    public function __construct(?int $storageSize_GB = null)
+    {
+        $this->storageSize_GB = $storageSize_GB;
+    }
+
+    public static function fromArray(array $data): self
+    {
+        return new self(
+            $data['storageSize_GB'] ?? null,
+        );
+    }
+
+    public function toArray(): array
+    {
+        $data = [];
+
+        if ($this->storageSize_GB !== null) {
+            $data['storageSize_GB'] = $this->storageSize_GB;
+        }
+
+        return $data;
+    }
+
+    public function getStorageSizeGB(): ?int
+    {
+        return $this->storageSize_GB;
+    }
+
+    public function setStorageSizeGB(?int $storageSize_GB): void
```

Comment:
ok, maybe I can understand making the parent nullable (in order to delete it), but why would we ever set this value as null?